### PR TITLE
data: re-add snapd.refresh.{timer,service} with weekly schedule

### PR DESCRIPTION
**Pull request by mvo5**
This will protect us against bugs in the new internal-refresh
code. If for some reason this fails we will still use a systemd
timer to refresh once a week.

### CODE REVIEW

File: data/systemd/snapd.refresh.service [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+Description=Automatically refresh installed snaps
+After=network-online.target snapd.socket
+Requires=snapd.socket
+ConditionPathExistsGlob=/snap/*/current
+Documentation=man:snap(1)
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/snap refresh
+Environment=SNAP_REFRESH_FROM_EMERGENRY_TIMER=1
```
Comment (pedronis) [inline data/systemd/snapd.refresh.service]: typo EMERGENCY

**Comment by pedronis** [comment]
given that is random, don't we need to stop it during tests?

**Comment by mvo5** [comment]
@pedronis Indeed, I disabled the timer in the tests now too

File: cmd/snap-confine/spread-tests/regression/lp-1599608/task.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ prepare: |
     cd /
     echo "Install hello-world"
     snap install hello-world
-    systemctl stop snapd.service snapd.socket
+    systemctl stop snapd.refresh.timer snapd.service snapd.socket
```
Comment (pedronis) [inline cmd/snap-confine/spread-tests/regression/lp-1599608/task.yaml]: do we need this given is disabled in the project?

Comment (mvo5) [inline cmd/snap-confine/spread-tests/regression/lp-1599608/task.yaml]: Indeed, I think we do not really need it, I will remove (but waiting for tests first).

**Pull request merged**